From b9fdbdf20d62655c9b529f744f8efb9fb66c5851 Mon Sep 17 00:00:00 2001
From: Scott Little <scott.little@windriver.com>
Date: Wed, 25 Oct 2017 16:13:20 -0400
Subject: [PATCH] Modify error code of
 bz1454699-LVM-status-check-for-missing-VG.patch to prevent controler-1 reboot
 loop

---
 heartbeat/LVM | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/heartbeat/LVM b/heartbeat/LVM
index 5347765..e4cd0ea 100755
--- a/heartbeat/LVM
+++ b/heartbeat/LVM
@@ -348,7 +348,7 @@ LVM_status() {
 	fi
 	if ! echo "$output" | grep -q "Found.*\"$1\""; then
 		ocf_exit_reason "LVM Volume $1 is not available"
-		return $OCF_ERR_GENERIC
+		return $OCF_NOT_RUNNING
 	fi
 
 	# Ask lvm whether the volume group is active.  This maps to
-- 
1.9.1

